src/CssInliner.php

```diff
@@ -586,7 +630,23 @@ private function collateCssRules(CssDocument $parsedCss): array
 
             $mediaQuery = $cssRule->getContainingAtRule();
             $declarationsBlock = $cssRule->getDeclarationAsText();
-            foreach ($cssRule->getSelectors() as $selector) {
+            $selectors = $cssRule->getSelectors();
+
+            // Maybe exclude CSS selectors
+            if (!empty($this->excludedCssSelectors)) {
+                // Normalize spaces, line breaks & tabs
+                $selectorsNormalized = \array_map(static function ($selector) {
+                    $selector = \str_replace(["\r", "\n", "\t"], ' ', $selector);
+                    $selector = \preg_replace('@[ ' . self::ALL_SPACES . ']+@mu', ' ', $selector);
```

Comment:
Surely you can just use `\s++` in the regular expression to match all whitespace sequences (without having to reinvent the wheel, so to speak) - ?

(Also, you don't need the `m` modifier as you are not matching `.` (any character) at all.)

Reply:
No problem, regex are not my strong suit, I changed it. Let me know if you see any other issues.